agent/dockerclient/dockerauth/ecr.go

```diff
@@ -65,9 +66,16 @@ func NewECRAuthProvider(ecrFactory ecr.ECRFactory, cache async.Cache) DockerAuth
 
 // GetAuthconfig retrieves the correct auth configuration for the given repository
 func (authProvider *ecrAuthProvider) GetAuthconfig(image string,
-	authData *ecr.ECRAuthData) (docker.AuthConfiguration, error) {
+	registryAuthData *api.RegistryAuthenticationData) (docker.AuthConfiguration, error) {
+
+	if registryAuthData == nil {
+		return docker.AuthConfiguration{}, fmt.Errorf("dockerauth: missing container's registry auth data")
```

Comment:
Should this be "container registry's"? Or just "missing container registry auth data" (no possessive)?